### ISSUE TITLE
build Appimage for Debian

### ISSUE THREAD

**Issue by chrisWith** [bkueng/qMasterPassword#7]
Hello,
it would be great if I could use this app also for my linux debian distro by making this as an appimage for example. This would be helpful for almost every linux distro.

Here is the link how to build appimages:
 https://github.com/AppImage/AppImageKit

The developer made also a video on how to make it:
https://www.youtube.com/watch?v=UJJKnr-WO0Y

best regards
chris

**Comment by probonopd** [comment]
@bkueng please let me know if you would like to have support implementing this

**Comment by bkueng** [comment]
Thanks @probonopd, it's simple enough with a nice example script so that I quickly did it myself: https://github.com/bkueng/qMasterPassword/commit/01b178c124c0a6c6ba3a71af7746d1a61ad6676c.

Here's the binary from master: https://drive.google.com/file/d/1LiaVX8WRrOXZtzkxtb0xla5Qo78e6E5X/view?usp=sharing
Let me know if it works.

**Comment by chrisWith** [comment]
Great. Right now I'm on vacation until next week. When I'm back I try it. Thank you in advance. Mon Jan 13 19:05:01 GMT+01:00 2020 bkueng/qMasterPassword <reply@reply.github.com>: Thanks @probonopd, it's simple enough with a nice example script so that I quickly did it myself: 01b178c. Here's the binary from master: https://drive.google.com/file/d/1LiaVX8WRrOXZtzkxtb0xla5Qo78e6E5X/view?usp=sharing Let me know if it works. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by probonopd** [comment]
Thanks @bkueng. Would it be possible to compile this on the oldest system it can be compiled on, e.g., Ubuntu xenial? Right now it is not working e.g., on Ubuntu 18.04 because it was compiled on a system newer than that.

**Comment by bkueng** [comment]
Ah, I was not aware of that. Is it the libc version that matters?
I'll look into it.

**Comment by probonopd** [comment]
Yes, libc and any other dependency lib. You can use a system like Travis CI if you don't want to install another OS version on your local machine. Check out the example at https://github.com/probonopd/linuxdeployqt#using-linuxdeployqt-with-travis-ci.

**Comment by chrisWith** [comment]
Hi together, I did a test in ArchLinux and it works fine. But on MX-Linux (Debian) and  Solus-Linux it did not start.

**Comment by bkueng** [comment]
This one is built with Ubuntu 14.04 (via docker): https://drive.google.com/open?id=11K9W33GdMbjqVI1ZXrWGnm-Iylxco3Cd

**Comment by probonopd** [comment]
Runs for me on Ubuntu 18.04.
